docker_dyno_driver.go

```diff
@@ -96,6 +98,11 @@ func (dd *DockerDynoDriver) Start(ex *Executor) error {
 		log.Fatal(err)
 	}
 
+	ex.container, err = dd.d.c.InspectContainer(container.ID)
```

Comment:
@fabiokung i'm finding the container isn't always given an IP address. it's not consistent behavior, but it happens once in a few `ftest-docker` runs. i haven't found a predictable set of checks or delays between checks that ensure I get an IP address. an infinite loop seems to work, but that doesn't feel right.